dev/com.ibm.ws.ras.instrument_test/test/com/ibm/example/bytecode/HelloWorldJava11.java

```diff
@@ -0,0 +1,21 @@
+package com.ibm.example.bytecode;
```

Comment:
Add Local variable syntax for lambda parameters -> https://openjdk.org/jeps/323?